src/main/java/reactor/core/publisher/Mono.java

```diff
@@ -2517,6 +2515,25 @@ public String toString() {
 	}
 
 	/**
+	 * Transform this {@link Mono} in order to generate a target {@link Mono}. Unlike {@link #compose(Function)}, the
+	 * provided function is executed as part of assembly.
+	 *
+	 * {@code Function<Mono, Mono> applySchedulers = mono -> mono.subscribeOn(Schedulers.io()).publishOn(Schedulers.computation());
+	 *        mono.transform(applySchedulers).map(v -> v * v).subscribe(Subscribers.unbounded())}
+	 *
+	 * @param transformer the {@link Function} to immediately map this {@link Mono} into a target {@link Mono}
+	 * instance.
+	 * @param <V> the item type in the returned {@link Mono}
+	 *
+	 * @return a new {@link Mono}
+	 * @see #compose(Function) for deferred composition of {@link Mono} for each {@link Subscriber}
+	 * @see #as for a loose conversion to an arbitrary type
+	 */
+	public final <V> Mono<V> transform(Function<? super Mono<T>, ? extends Mono<V>> transformer) {
```

Comment:
No, why?  This isn't loose `as()` but rather designed for Mono chaining in the same way composed is used but without deferring the transformation.  What i didn't update here (but did in compose) is the restriction in function type. Should be:
`public final <V> Mono<V> transform(Function<? super Mono<T>, ? extends Publisher<V>> transformer)`